tests/zfs-tests/tests/functional/write_dirs/setup.ksh

```diff
@@ -51,4 +51,7 @@ DISK=${DISKS%% *}
 
 log_must set_partition $SLICE "" $SIZE $DISK
 
+if is_linux; then
+	export SLICE=1
+fi
```

Comment:
Originally this was only to make `write_dirs/setup` succeed with a real disk and `write_dirs_001_pos` to pass.
I've just tried `write_dirs_002_pos` and it caused the following:
```
[335174.862685] Kernel panic - not syncing: hung_task: blocked tasks
```
So, no change here.